ceph-pr-combined/config/definitions/ceph-pr-combined.yml

```diff
@@ -0,0 +1,135 @@
+- job:
+    name: ceph-pr-combined
+    description: 'make check and ceph API tests combined'
+    project-type: freestyle
+    defaults: global
+    concurrent: true
+    node: huge && bionic && x86_64 && !xenial && !trusty
+    display-name: 'ceph PRs: make check and API tests'
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - build-discarder:
+          days-to-keep: 15
+          num-to-keep: 300
+          artifact-days-to-keep: -1
+          artifact-num-to-keep: -1
+      - github:
+          url: https://github.com/ceph/ceph/
+      - rebuild:
+          auto-rebuild: true
+      - inject:
+          properties-content: |
+            TERM=xterm
+
+    parameters:
+      - string:
+          name: ghprbPullId
+          description: "the GitHub pull id, like '72' in 'ceph/pull/72'"
+
+    triggers:
+      - github-pull-request:
+          cancel-builds-on-update: true
+          allow-whitelist-orgs-as-admins: true
+          org-list:
+            - ceph
+          white-list-target-branches:
+            - master
+            - octopus
+            - nautilus
+          trigger-phrase: '^jenkins test make check|^jenkins test pr.*|^jenkins retest.*'
```

Comment:
There's no way to just re-trigger the API tests from artifacts/reusing the current workspace, right? We may keep the `jenkins test api` just for 'backward compatibility'.

Reply:
We can for sure keep "jenkins test api" for backwards compatibility.  And, no, the slave would have to sit around with the binaries created by `make` in order to *just* retest the API.